robottelo/hosts.py

```diff
@@ -2461,6 +2461,23 @@ def run_orphan_cleanup(self, smart_proxy_id=None):
             max_tries=10,
         )
 
+    def run_repos_refresh(self):
+        """Run repo refresh rake task"""
+        timestamp = datetime.now(UTC).replace(microsecond=0)
+        self.execute('foreman-rake katello:refresh_repos')
+        self.wait_for_tasks(
+            search_query=(
+                'label = Actions::Pulp3::Orchestration::Repository::RefreshRepos'
+                f' and started_at >= "{timestamp}"'
+            ),
+            search_rate=5,
+            max_tries=10,
+        )
+
+    def set_pulp_cli_safemode(self, safe):
```

Comment:
is updating the config file really enough for the change to take effect (e.g. no service reload/restart required)? 🤔